backend/de.metas.handlingunits.base/src/main/sql/postgresql/system/5559900_sys_gh6713_add_warehouse_from_and_to_on_DD_Order.sql

```diff
@@ -0,0 +1,130 @@
+;
+
+-- 2020-05-25T08:11:46.128Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */  select update_Column_Translation_From_AD_Element(577736) 
+;
+
+-- 2020-05-25T08:11:58.930Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */ SELECT public.db_alter_table('DD_Order','ALTER TABLE public.DD_Order ADD COLUMN M_Warehouse_From_ID NUMERIC(10) DEFAULT 540008 NOT NULL')
+;
+
+-- 2020-05-25T08:11:59.850Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+ALTER TABLE DD_Order ADD CONSTRAINT MWarehouseFrom_DDOrder FOREIGN KEY (M_Warehouse_From_ID) REFERENCES public.M_Warehouse DEFERRABLE INITIALLY DEFERRED
+;
+
+-- 2020-05-25T08:13:04.894Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Reference_Value_ID,AD_Table_ID,ColumnName,Created,CreatedBy,DDL_NoForeignKey,DefaultValue,EntityType,FacetFilterSeqNo,FieldLength,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutoApplyValidationRule,IsAutocomplete,IsCalculated,IsDimension,IsDLMPartitionBoundary,IsEncrypted,IsFacetFilter,IsForceIncludeInGeneratedModel,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsRangeFilter,IsSelectionColumn,IsShowFilterIncrementButtons,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,MaxFacetsToFetch,Name,ReadOnlyLogic,SelectionColumnSeqNo,SeqNo,Updated,UpdatedBy,Version) VALUES (0,570820,577737,0,30,540420,53037,'M_Warehouse_To_ID',TO_TIMESTAMP('2020-05-25 11:13:04','YYYY-MM-DD HH24:MI:SS'),100,'N','540005','EE01',0,10,'Y','N','Y','N','N','N','N','N','N','N','N','N','N','N','N','N','N','Y','N','N','N','N','N','N','N','Y','N',0,'Warehouse to','',0,0,TO_TIMESTAMP('2020-05-25 11:13:04','YYYY-MM-DD HH24:MI:SS'),100,0)
+;
+
+-- 2020-05-25T08:13:04.897Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column_Trl (AD_Language,AD_Column_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language, t.AD_Column_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Column t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N') AND t.AD_Column_ID=570820 AND NOT EXISTS (SELECT 1 FROM AD_Column_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Column_ID=t.AD_Column_ID)
+;
+
+-- 2020-05-25T08:13:04.899Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */  select update_Column_Translation_From_AD_Element(577737) 
+;
+
+-- 2020-05-25T08:13:29.659Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+/* DDL */ SELECT public.db_alter_table('DD_Order','ALTER TABLE public.DD_Order ADD COLUMN M_Warehouse_To_ID NUMERIC(10) DEFAULT 540005 NOT NULL')
```

Comment:
who is M_Wareohuse_ID=540005?
Can we avoid hardcoding it?

Reply:
`Lager für Streckengeschäft` the default value for `M_Warehouse_To_ID` agreed upon with Mark. I need this to be mandatory in order to create `DD_OrderLines` via quick input.